### DISCUSSION TITLE
Ant Design 全球首次开源 Meetup 分享

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#875] [General]
Ant Design 社区会在 5 月 28 号在杭州有一个 Meetup，Ant Design Web3 也会有一个 30min 的分享，欢迎大家提建议，看看聊点什么。

**Comment by yutingzhao1991** [comment]
初步讨论后目前定的议题是：

《从 antd 中来，到 web3 中去 —— 我和 Ant Design Web3 的故事》
- Ant Design Web3 简介  5min
- Ant Design Web3 带我进入开源 10min
- 我看炒币那些事 5min
- Web3 和开源的想象力 5min

主讲人 @gin-lsl(曾缺) 。

**Comment by yutingzhao1991** [comment]
《Ant Design Web3 简介》这部分我来讲，开个头：

Ant Design Web3 是什么？

基于 Ant Design 的一个组件库，有点类似 Ant Design Pro Components 这样的形式。

比如 ConnectModal、Address、NFTCard。【展示组件 UI】

但是又不完全类似，因为它不是纯粹的 UI 层，还包含了部分和区块链相关的内容，除了对 UI 的封装外还有对链交互的封装。

实际上，我们最后包含了好几个 npm 包：

- web3
- web3-wagmi
- web3-solana
- web3-bitcoin

其中 web3 这个包是纯 UI 的，但是 balabala……

以上是对 Ant Design Web3 的介绍，但是今天这个场子，我不是来推广 Ant Design Web3 的，更重要的是来和大家交流开源建设的一些心得体会，参与开源有时候会有一些商业考量，有时候是为了学习，有时候就是纯粹喜欢，有时候是为了交朋友，所以我想要把时间更多的交给参与我们开源建设的同学，让他们来分享参与到 Ant Design Web3 建设中的感受，也分享下各自对 Web3 对加密货币的看法。

接下来请 Ant Design Web3 的核心贡献者，也是 web3-solana 的贡献者，来自郑州的曾缺上台分享。

@gin-lsl 我大概会以这样的形式开场。

@thinkasany @kiner-tang @jeasonstudio 你们要是感兴趣都可以上来讲讲，只要我们整体时间控制好就行。

**Reply by thinkasany** [reply]
到时候是以ppt还是yuque文章也行呢？ 我的话可以先准备关于“《开源项目的 CI 漫谈》”，到时候需要上就上，不用的话就当来玩和复盘了。 讲讲我们项目中已经应用的ci，分享一些已经在用的或者往期踩坑经验吧。

**Reply by yutingzhao1991** [reply]
有 PPT

**Reply by yutingzhao1991** [reply]
《开源项目的 CI 漫谈》这个感觉嵌入到我们的分享中有点奇怪，不过我可以问下组委会有没有什么圆桌啥的可以邀请社区同学更深入参与进来交流这些开源建设的点

**Comment by yutingzhao1991** [comment]
Web3 和开源的想象力 5min 这部分 @jeasonstudio 会来讲

**Comment by jeasonstudio** [comment]
Web3 + 开源想象力：
* DApp 开发的现状和不足
* 通过 `connect` 打通链上链下
* 基于以太坊的多链架构
* 深入 Web3 社区

**Comment by xuiang** [comment]
hello~请问ant-design有微信区或者其他社坛吗？

**Reply by gin-lsl** [reply]
首页有 Telegram 链接，不过其实 GitHub 更方便点。

**Reply by thinkasany** [reply]
钉钉群可以拉一下，不过我的账号现在还是封禁状态可能你们看不见我的评论。

**Reply by thinkasany** [reply]
<img src="https://github.com/ant-design/ant-design-web3/assets/117748716/4427c32b-c73b-4ae2-b7d9-3b0a5142e4ea" alt="Ant Design Web3 Community Group QCcode" height="200" />
